core/system/rule_manager_test.go

```diff
@@ -1,16 +1,128 @@
 package system
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
+func TestGetRules(t *testing.T) {
+	t.Run("EmptyRules", func(t *testing.T) {
+		rules := GetRules()
+		assert.Equal(t, len(rules), 0)
+	})
+
+	t.Run("GetUpdatedRules", func(t *testing.T) {
+		sRule := []*SystemRule{
+			{MetricType: InboundQPS, TriggerCount: 1},
+			{MetricType: Concurrency, TriggerCount: 2},
+		}
+		onRuleUpdate(sRule)
```

Comment:
This suggestion is very good. 
And I add 
    `defer func() { ruleMap = make(RuleMap, 0) }()`
if ruleMap is changed, to avoid the impact between different tests.

Reply:
Yes, your advice is very nice.